pages/create-account.js

```diff
@@ -0,0 +1,143 @@
+import Head from 'next/head';
+import React, { useState } from 'react';
+import styles from '../styles/Home.module.css';
+
+export default function Account() {
+  const [errors, setErrors] = useState({
```

Comment:
These state fields aren't errors, rename accordingly.

```suggestion
  const [inputFields, setInputFields] = useState({
```